frontend/src/features/properties/map/acquisition/add/AddAcquisitionFileYupSchema.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-template-curly-in-string */
+import * as Yup from 'yup';
+
+export const AddAcquisitionFileYupSchema = Yup.object().shape({
+  name: Yup.string()
+    .required('Acquisition file name is required')
+    .max(500, 'Acquisition file name must be at most ${max} characters'),
```

Comment:
this is the same as saying "blah must be at most 500 chars" but with advantage that if we ever change the limit we don't have to update the message as well. Seems useful